README.md

```diff
@@ -15,6 +15,19 @@ To see Codalab Competition's in action, visit [www.competitions.codalab.org](htt
 The CodaLab community forum is hosted on Google Groups.
 - [CodaLabDev Google Groups Forum](https://groups.google.com/forum/#!forum/codalabdev)
 
+## Compute worker
+
+To start the compute worker under project directory:
+
+`docker-compose up worker`
```

Comment:
How long would it take to modify this to run everything -- django, compute worker, site worker, celery, and mysql? Could we run compute worker independently still with that kind of setup?